CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 ### Added
+- Update Back links with introduction of new class, styles and icon [#799](https://github.com/hmrc/assets-frontend/pull/799)
+
+### Added
```

Comment:
There should only be one of each section. Can you consolidate the Added sections please?